pycon/dev-settings.py

```diff
@@ -26,12 +26,15 @@
 
 TEMPLATES[0]['OPTIONS']['debug'] = True
 
-INSTALLED_APPS = INSTALLED_APPS + (
+# The cast to set to tuple is a workaround allowing to have `django_extensions`
```

Comment:
not sure about this. especially about this `tuple()` maybe use `list()` instead? or just make sure that those apps are not duplicated?

Reply:
`INSTALLED_APPS` is a `tuple` in `pycon.settings`, that's why I kept it here. I agree the change is controversial - I'll remove this change for now as it's something that's going to require discussion and I wouldn't want it to delay merging this PR.